objects/src/assets/mod.rs

```diff
@@ -0,0 +1,27 @@
+use super::{Felt, Word};
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum Asset {
+    Fungible(AssetId, u64),
```

Comment:
Consider having named fields for variants that contains more than one inner attribute. Example:

`Fungible { id: AssetId, amount: u64 }`

This way, it's easier to immediately spot what each of the attributes is.

Reply:
Ah - I checked in this file by mistake. But I'll add these in the next PR.